src/mds/MDSMap.cc

```diff
@@ -221,6 +221,7 @@ void MDSMap::dump(Formatter *f) const
   f->close_section();
   f->dump_int("metadata_pool", metadata_pool);
   f->dump_bool("enabled", enabled);
+  f->dump_bool("prevent_client_session", prevent_client_session);
```

Comment:
How about `s/prevent_client_session/reject_client_reconnect/` ? I am not good at naming this.

Reply:
'reconnect' would be a bit misleading here I feel. Maybe `reject_client_session` can be done or even `reject_client_connect`? What do you think?

Reply:
+1 for `reject_client_session`.

Reply:
Thanks @lxbsz @vshankar, I've pushed the change. PTAL.